config/servicemappings/privilegedaccessmanager.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: core.cnrm.cloud.google.com/v1alpha1
+kind: ServiceMapping
+metadata:
+  name: privilegedaccessmanager.cnrm.cloud.google.com
+  namespace: cnrm-system
+spec:
+  name: PrivilegedAccessManager
+  version: v1alpha1
+  serviceHostName: "privilegedaccessmanager.googleapis.com"
+  resources:
+    - name: google_privileged_access_manager_entitlement
+      kind: PrivilegedAccessManagerEntitlement
+      direct: true
```

Comment:
I am curious why this file is needed? We don't have a service mapping for workstations currently.

Reply:
Good point! I thought it is required for `make ready-pr` but it turned out to be not necessary so removed it in the diffbase (#2596).